tests/e2e/test_pv_scale_ocs_create_delete_pvcs.py

```diff
@@ -0,0 +1,184 @@
+"""
+PV Create/Delete & Memory Leak Test: Test the PVC limit with 3 worker nodes
+with & without any IO create and delete the PVCs and check for memory leak
+TO DO: This Test needs to be executed in Scaled setup,
+Adding node scale is yet to be supported.
+"""
+import logging
+import pytest
+import random
+import time
+
+from concurrent.futures import ThreadPoolExecutor
+from tests import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import pod, pvc
+from ocs_ci.framework.testlib import scale, E2ETest
+
+log = logging.getLogger(__name__)
+
+
+class BasePvcPodCreateDelete(E2ETest):
+    """
+    Base Class to create/delete PVC and POD
+    """
+
+    def create_pvc_pod(self, rbd_sc_obj, cephfs_sc_obj, number_of_pvc, size, start_io):
```

Comment:
will it make sense to have cephfs and rbd tested in separate test cases?

Reply:
The requirement of the Test case is to have all the PVC types and pods